services/payments/tests/unit/conftest.py

```diff
@@ -395,6 +397,113 @@ def mock_payments_gateway_service_or_none(
     return mock_payments_gateway_service_api_base
 
 
+#
+# mock Stripe API
+#
+
+
+@pytest.fixture
+def mock_payments_stripe_api_base(app: FastAPI) -> Iterator[MockRouter]:
+    """
+    If external_environment is present, then this mock is not really used
+    and instead the test runs against some real services
+    """
+    settings: ApplicationSettings = app.state.settings
+
+    with respx.mock(
+        base_url=settings.PAYMENTS_STRIPE_URL,
+        assert_all_called=False,
+        assert_all_mocked=True,  # IMPORTANT: KEEP always True!
+    ) as respx_mock:
+        yield respx_mock
+
+
+@pytest.fixture
+def mock_payments_stripe_routes(faker: Faker) -> Callable:
+    def _mock(mock_router: MockRouter):
+        def _list_products(request: httpx.Request):
```

Comment:
MINOR: doc with e.g. a link where the API specs are in case we want to textend
Q: isnt there an automatic strip mock? Similar to moto for s3? Found https://github.com/stripe/stripe-mock

Reply:
Added API docs for now :+1: Good catch with the stripe-mock; it didn't come to my mind. Yes, we can also start to use it if we will start to use the Stripe API more.